book/tables/ECG/egt01.qmd

```diff
@@ -83,7 +83,9 @@ result
 
 ## `teal` App
 
-```{r teal, opts.label=c('skip_if_testing', 'app')}
+```{r teal, opts.label=c('skip_if_testing', 'app'), eval = packageVersion("teal.modules.clinical") >= "0.9.0.9007"}
```

Comment:
really!? can you do that!? specify different versions?

Reply:
of course :) this means that a given chunk (that is: teal app) won't be evaluated if condition is not met. The code will be still there but no screenshot. Similar thing can be done for non-teal code chunks.